cmake/BLTMacros.cmake

```diff
@@ -1274,96 +1274,92 @@ endmacro(blt_combine_static_libraries)
 
 
 ##------------------------------------------------------------------------------
-## blt_print_target_properties(TARGET <target> )
+## blt_print_target_properties(TARGET <target> CHILDREN <bool> PROP_REGEX <regular_expression_string>)
 ##
-## Prints out all properties of the given target.
+## Prints out properties of the given target.
+##
+## Has the options to print target's link libraries and interface link libraries
+## with the CHILDREN argument, as well as specific properties using a regular
+## expression.
+## 
+## Defaults:
+## CHILDREN = false (non recursive)
+## PROP_REGEX = .* (print every property)
```

Comment:
Can you change this to default to an empty string and skip the regex matching completely if its empty?  Regex's are heavy handed and should be avoided if possible.  Not that speed super matters in this case but we should still optimize when its easy.

Reply:
I did several tests and it ended up being exactly the same.. so you can leave it in.